src/base/types/Reaction.jl

```diff
@@ -1,6 +1,7 @@
 """
     mutable struct Reaction
         id::String
+        name::Maybe{String}
```

Comment:
note to self: names should also be supported by SBML models (SBML.jl supports names now)

Reply:
in #568